nix/nim-status/status-go.nix

```diff
@@ -0,0 +1,97 @@
+{ stdenv, lib, fetchFromGitHub, buildGo114Package
+# Dependencies
+, xcodeWrapper
+, go, androidPkgs
+# metadata and status-go source
+, meta, source
+, newScope
+# build parameters
+, platform ? "android"
+, arch ? "386"
+, api ? "23" }:
+
+let
+  inherit (lib) attrNames attrValues getAttr mapAttrs strings concatStringsSep concatMapStrings;
+
+  callPackage = newScope {};
+  flags = callPackage ./getFlags.nix {inherit platform arch;};
+
+  removeReferences = [ go ];
+  removeExpr = refs: ''remove-references-to ${concatMapStrings (ref: " -t ${ref}") refs}'';
+
+  # Params to be set at build time, important for About section and metrics
+  goBuildParams = {
+    GitCommit = source.rev;
+    Version = source.cleanVersion;
+  };
```

Comment:
This is not being used. Why?
```
 > nix-linter nix/nim-status/status-go.nix    
Unused `let` bind `goBuildParams` at nix/nim-status/status-go.nix:23:3
Unused argument `stdenv` at nix/nim-status/status-go.nix:1:1-98:1
Unused argument `fetchFromGitHub` at nix/nim-status/status-go.nix:1:1-98:1
Unused argument `xcodeWrapper` at nix/nim-status/status-go.nix:1:1-98:1
Unused argument `meta` at nix/nim-status/status-go.nix:1:1-98:1
```

Reply:
Went through the arguments to Nix packages, removed unused ones.